ha/monitor/k8s/parser.py

```diff
@@ -24,18 +24,18 @@
 from ha.monitor.k8s.const import EventStates
 
 from cortx.utils.log import Log
-from ha.fault_tolerance.event import Event
-from ha.fault_tolerance.const import HEALTH_ATTRIBUTES, HEALTH_EVENT_SOURCES
+from ha.fault_tolerance.const import HEALTH_EVENT_SOURCES
+from cortx.utils.event_framework.health import HealthAttr, HealthEvent
 
 
 class ObjectParser:
     def __init__(self):
-        self.payload = {HEALTH_ATTRIBUTES.SOURCE.value: HEALTH_EVENT_SOURCES.MONITOR.value, \
-                    HEALTH_ATTRIBUTES.CLUSTER_ID.value: None, HEALTH_ATTRIBUTES.SITE_ID.value: None, \
-                    HEALTH_ATTRIBUTES.RACK_ID.value: None, HEALTH_ATTRIBUTES.STORAGESET_ID.value: None, \
-                    HEALTH_ATTRIBUTES.NODE_ID.value: None, HEALTH_ATTRIBUTES.RESOURCE_TYPE.value: None, \
-                    HEALTH_ATTRIBUTES.RESOURCE_ID.value: None, HEALTH_ATTRIBUTES.RESOURCE_STATUS.value: None, \
-                    HEALTH_ATTRIBUTES.SPECIFIC_INFO.value: {}}
+        self.payload = {HealthAttr.SOURCE.value: HEALTH_EVENT_SOURCES.MONITOR.value,
+                    HealthAttr.CLUSTER_ID.value: None, HealthAttr.SITE_ID.value: None,
```

Comment:
@gauri-bhosale Can we use the NOT_DEFINED constant here for site, rack and storageset_id?

Reply:
Added NOT_DEFINED to site , rack and storageset_id